### ISSUE TITLE
Our Map guide page is a terrible introduction to Map

### ISSUE THREAD

**Issue by Cadair** [sunpy/sunpy#5573]
https://docs.sunpy.org/en/stable/guide/data_types/maps.html

If you haven't ever used Map before (maybe have minimal background with solar data) this is a terrible introduction. Sure the information about making a custom map is useful somewhere in our docs, but two paragraphs in at the top of the main guide page?! No.

**Comment by mwhv2** [comment]
As someone who has been using the guides and docs a lot for the past several months, I'd be happy to edit the Map introduction while keeping the points you listed in mind (if no one else is working on this).

**Comment by nabobalis** [comment]
Go ahead, I don't think anyone is working on it.

**Comment by Cadair** [comment]
Yes please, you are very well placed to write docs, you have just about the perfect level of familiarity and recent trauma from reading our current docs! :stuck_out_tongue:

**Comment by dstansby** [comment]
Updated content is now at https://docs.sunpy.org/en/latest/tutorial/maps.html - would be good for you to take a look at it @Cadair and decide if it's good enough to close this issue.
